spec/namespaces/_core.yaml

```diff
@@ -2837,11 +2841,12 @@ components:
               creation_time:
                 type: integer
                 format: int64
-    delete@200:
+    delete@200: &WriteResponseBase
```

Comment:
This is an interesting syntax! Where can I read about `&` and `*`?

Reply:
Thanks. 
I referred these resources before implementing:

1. https://yaml.org/spec/1.2.2/#692-node-anchors
2. https://support.atlassian.com/bitbucket-cloud/docs/yaml-anchors/